assembl/static2/js/app/components/administration/voteSession/pageForm.jsx

```diff
@@ -0,0 +1,154 @@
+// @flow
+import React from 'react';
+import { connect } from 'react-redux';
+import { I18n } from 'react-redux-i18n';
+import { FormGroup } from 'react-bootstrap';
+import SectionTitle from '../sectionTitle';
+import FormControlWithLabel from '../../common/formControlWithLabel';
+import FileUploader from '../../common/fileUploader';
+import TitleWithHelper from '../../common/titleWithHelper';
+import { getEntryValueForLocale } from '../../../utils/i18n';
+import {
+  updateVoteSessionPageTitle,
+  updateVoteSessionPageSubtitle,
+  updateVoteSessionHeaderImage,
+  updateVoteSessionPageInstructionsTitle,
+  updateVoteSessionPageInstructionsContent,
+  updateVoteSessionPagePropositionsTitle
+} from '../../../actions/adminActions/voteSession';
+
+type PageFormProps = {
+  headerTitle: string,
+  headerSubtitle: string,
+  headerImgUrl: string,
+  headerImgMimeType: string,
+  instructionsTitle: string,
+  instructionsContent: string,
+  propositionSectionTitle: string,
+  selectedLocale: string,
+  handleHeaderTitleChange: Function,
+  handleHeaderSubtitleChange: Function,
+  handleHeaderImageChange: Function,
+  handleInstructionsTitleChange: Function,
+  handleInstructionsContentChange: Function,
+  handlePropositionSectionTitleChange: Function
+};
+
+const PageForm = ({
+  headerTitle,
+  headerSubtitle,
+  headerImgUrl,
+  headerImgMimeType,
+  instructionsTitle,
+  instructionsContent,
+  propositionSectionTitle,
+  selectedLocale,
+  handleHeaderTitleChange,
+  handleHeaderSubtitleChange,
+  handleHeaderImageChange,
+  handleInstructionsTitleChange,
+  handleInstructionsContentChange,
+  handlePropositionSectionTitleChange
+}: PageFormProps) => {
+  const headerTitlePh = `${I18n.t('administration.ph.headerTitle')} ${selectedLocale.toUpperCase()}*`;
+  const headerSubtitlePh = `${I18n.t('administration.ph.headerSubtitle')} ${selectedLocale.toUpperCase()}`;
+  const instructionsTitlePh = `${I18n.t('administration.ph.instructionsTitle')} ${selectedLocale.toUpperCase()}*`;
+  const instructionsContentPh = `${I18n.t('administration.ph.instructionsContent')} ${selectedLocale.toUpperCase()}*`;
+  const propositionSectionTitlePh = `${I18n.t('administration.ph.propositionSectionTitle')} ${selectedLocale.toUpperCase()}*`;
```

Comment:
I think that a const for `selectedLocale.toUpperCase()` would be nice here.
Also, we should look at how we can handle the style of required fields via css to avoid these `*` in strings/labels

Reply:
Pour l'étoile, il faudra ajouter sur les champs : `classeName={required ? "required" : ""}` puis dans le CSS faire
`required {
    &:after {
        content: "*";
    }
}`
Le problème c'est que je me casse les dents pour le mettre sur le placeholder du richtext.

Reply:
Pour moi ça doit se faire à l'intérieur du composant, ajouter la class required si t'as passé la prop required qu'on a déjà